session/bootstrap.go

```diff
@@ -357,6 +357,14 @@ const (
 		last_analyzed_at TIMESTAMP,
 		PRIMARY KEY (table_id, column_id) CLUSTERED
 	);`
+	// CreateTableCacheMetaTable stores the cached table meta lock information.
+	CreateTableCacheMetaTable = `CREATE TABLE IF NOT EXISTS mysql.table_cache_meta (
+		tid int(11) NOT NULL DEFAULT 0,
```

Comment:
```suggestion
		tid bigint(11) NOT NULL DEFAULT 0,
```

The other system tables use `BIGINT` type for table_id, should we keep them consistent?

Reply:
Later BR may need to support specifying table_id explicitly, I am not sure if it will be a problem.

Reply:
OK